sdks/java/core/src/main/java/org/apache/beam/sdk/io/BigQueryIO.java

```diff
@@ -1048,6 +1062,22 @@ public PDone apply(PCollection<TableRow> input) {
         return VoidCoder.of();
       }
 
+      @Override
+      public void populateDisplayData(DisplayData.Builder builder) {
+        super.populateDisplayData(builder);
+
+        if (tableRefFunction != null) {
+          builder.add("tableFn", tableRefFunction.getClass());
+        }
+
+        builder
+            .add("createDisposition", createDisposition.toString())
+            .add("writeDisposition", writeDisposition.toString())
+            .addIfNotNull("table", jsonTableRef)
+            .addIfNotNull("schema", jsonSchema)
```

Comment:
if ordering is or ever will be meaningful, I'd move table/schema to be the first two here.